### PR TITLE
Add export command

### PR DESCRIPTION
**Pull request by jarqvi**
**What I did**
I noticed that we have this command in the Docker CLI, but not in Docker Compose. So I thought of adding it to make it useful for others.

### CODE REVIEW

**Review by ndeloof — APPROVED** [review]
LGMT. Please sign-off your commit

**Comment by ndeloof** [comment]
not sure how you did, but better rebase on main branch, stash all your commit and amend with `--signoff`

**Comment by jarqvi** [comment]
> not sure how you did, but better rebase on main branch, stash all your commit and amend with `--signoff`

i use this command:
```
git commit --amend --no-edit --signoff
```

**Comment by jarqvi** [comment]
> not sure how you did, but better rebase on main branch, stash all your commit and amend with `--signoff`

Oh sorry, i solved this issue with a messy approach:)

**Comment by ndeloof** [comment]
Still sign-off line missing in commit message. Maybe amend and add it manually ?

**Comment by jarqvi** [comment]
> Still sign-off line missing in commit message. Maybe amend and add it manually ?

Done.
@ndeloof

**Review by ndeloof — APPROVED** [review]

**Review by glours — APPROVED** [review]
LGTM

**Comment by jarqvi** [comment]
Do I have to do anything else?

**Comment by ndeloof** [comment]
@jarqvi just need to fix issues reported by CI :)

**Comment by jarqvi** [comment]
@ndeloof  done =)

File: go.mod [hunk 1]
```diff
@@ -34,6 +34,7 @@ require (
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.1.0
 	github.com/otiai10/copy v1.14.0
+	github.com/pkg/errors v0.9.1
```
Comment (glours) [inline go.mod:37]: You should use `errors` and `fmt.Errorf` instead of the deprecated package `github.com/pkg/errors`

Comment (jarqvi) [reply go.mod]: ok, done.
@glours

**Comment by jarqvi** [comment]
> Sorry I rebased the branch to be sure we don't have a conflict, it changed the commit history and broke the DCO 🤦 Can you follow [those instructions](https://github.com/docker/compose/pull/12120/checks?check_run_id=30973008108) to fix the sign-off issues?

Done.
@glours

**Pull request merged**